### DISCUSSION TITLE
Added: Notes for November 4th, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4103] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-11-04.md

## Agenda

- Nullable parameter defaults
- Argument state after call for AllowNull parameters

**Comment by PathogenDavid** [comment]
> No particularly amusing quotes were said during this meeting, sorry.

This is extremely disappointing, but I appreciate the transparency.
